src/functions-and-arrays.js

```diff
@@ -1,21 +1,49 @@
 // Iteration #1: Find the maximum
-function maxOfTwoNumbers() {}
-
+function maxOfTwoNumbers(num1, num2) {
+if (num1 > num2) {
+  return num1;
+} else {
+  return num2
+}
+};
 
 
 // Iteration #2: Find longest word
 const words = ['mystery', 'brother', 'aviator', 'crocodile', 'pearl', 'orchard', 'crackpot'];
 
-function findLongestWord() {}
-
+function findLongestWord(arrWords) {
+
+  if (arrWords.length == 0){
+  return null;
+} else {
+  let longestWord = "";
+  for (let i = 0; i < arrWords.length; i++) {
+if (longestWord.length < arrWords[i].length) {
+  longestWord = arrWords[i];
+  return longestWord;
+//} else if (longestWord.length == longestWord.length) {
+//  return longestWord[1];
+}
+}
+}
+}
+//should return the longest occurrence when it has multiple words
 
 
 // Iteration #3: Calculate the sum
 const numbers = [6, 12, 1, 18, 13, 16, 2, 1, 8, 10];
 
-function sumNumbers() {}
-
-
+function sumNumbers(arrNumbers) {
+  let counter = 0;
+  if (arrNumbers.length == 0) {
+    return 0;
+  } else {
+  for (let i=0; i< arrNumbers.length; i++) {
+    return counter += arrNumbers[i];
+  }
+  }
+}
```

Comment:
- Same mistake here - your function does not return anything. 
- Any logic that is inside loop will be repeated each iteration. What you want is to update the variable. And only return it after you finish your full loop cycle

```
for (let i=0; i< arrNumbers.length; i++) {
    counter += arrNumbers[i];
  }
```

- add `return` at the end of else statement